recipes-kernel/linux/linux-mainline/0001-sun8i-h2-plus-orangepi-zero.dts-enable-ina219.patch

```diff
@@ -1,35 +0,0 @@
-From cf5094d34f61d9d51e6b4dd48c8b7a5a319c2b0b Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Tomasz=20=C5=BByjewski?= <tomasz.zyjewski@3mdeb.com>
-Date: Wed, 8 Apr 2020 09:01:05 +0200
-Subject: [PATCH] sun8i-h2-plus-orangepi-zero.dts: enable in219
-MIME-Version: 1.0
-Content-Type: text/plain; charset=UTF-8
-Content-Transfer-Encoding: 8bit
-
-Signed-off-by: Tomasz Żyjewski <tomasz.zyjewski@3mdeb.com>
----
- arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts | 9 +++++++++
- 1 file changed, 9 insertions(+)
-
-diff --git a/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts b/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
-index 84cd9c061227..3b80ec4f7435 100644
---- a/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
-+++ b/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
-@@ -132,6 +132,15 @@
- 	status = "okay";
- };
-
-+&i2c1 {
-+      ina219@40 {
-+              status = "okay";
```

Comment:
This patch is dropped, do we enable this device somewhere else?

Reply:
It is here: https://github.com/3mdeb/meta-rte/pull/73/files#diff-5b15643f99f3569abf46c083d1734f58498233cac6dae4e2ce4208d213bb0760